language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/rename/ProjectRenameTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This should be the year of the first commit. aka. 2021

Reply:
Updated. Thanks!